server/src/main/scala/cromwell/CromwellEntryPoint.scala

```diff
@@ -175,17 +176,24 @@ object CromwellEntryPoint extends GracefulStopSupport {
     import spray.json._
 
     val validation = args.validateSubmission(EntryPointLogger) map {
-      case ValidSubmission(w, u, r, i, o, l, z) =>
+      case ValidSubmission(w, u, r, i, o, l, z) =>{
```

Comment:
The `{` and `}` are redundant on a `case` (but maybe you want them to make things easier to read?)

Reply:
Yeah I like to add braces if it make things easier to read. But, here it does not matter as there is just one case! So, will remove it.